### DISCUSSION TITLE
Can I update 'showOffsets' dinamically?

### DISCUSSION THREAD

**Discussion by glaprida** [GrapesJS/grapesjs#5032] [Q&A] [answered]
I want to add a panel button to be able to toggle `showOffsets` similar to the `sw-visibility` command.
I know how to add a button panel and bind it to a custom command, but I'm not sure what this command should do.

Something like: `editor.set('showOffsets',true)`

Any help is really appreciated! Thanks!

**Comment by artf** [comment, marked as answer]
It's a matter of updating the config here `editor.getConfig().showOffsets = false`
